.ci_support/linux64.yaml

```diff
@@ -9,6 +9,6 @@ target_platform:
 channel_sources:
 - conda-forge
 docker_image:
-- quay.io/condaforge/linux-anvil-comp7
+- quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
can you undo these changes?

COS7 will be automatically requested I believe when you add teh right things as requirements in the recipe.

Reply:
I tried, but ```run_docker_build.sh``` is using ```DOCKER_IMAGE=$(cat .ci_support/${CONFIG}.yaml | grep '^docker_image:$' -A 1 | tail -n 1 | cut -b 3-)``` to pick the image and doesn't check if you have a ```sysroot_linux-64 2.17``` build requirement or not.

Reply:
You should definitely not have this here like wolfv suggested. Try adding a conda-forge.yml file next to meta.yaml with

```
os_version:
  linux_64: cos7
```

That should almost certainly do the trick.